src/nodeinfoprovider/nodeinfoprovider.cpp

```diff
@@ -47,6 +47,8 @@ Error NodeInfoProvider::Init()
     mNodeInfo.mNodeType = cNodeType;
     mNodeInfo.mMaxDMIPS = cMaxDMIPS;
 
+    mNodeInfo.mCPUs.PushBack(CPUInfo {});
```

Comment:
The root cause I think is that nanopb sets nil for empty containers. Unlike grpc proto generator, that has clear_<field>() method to set a field to nil.

Reply:
the root cause is cloud wants this field mandatory. nil is properly handled on out side.